identifiers/country-hk/geographical_constituencies.csv

```diff
@@ -1,6 +1,14 @@
-id,name
-ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong Island
-ocd-division/country:hk/geographical_constituency:kowloon_west,Kowloon West
-ocd-division/country:hk/geographical_constituency:kowloon_east,Kowloon East
-ocd-division/country:hk/geographical_constituency:new_territories_west,New Territories West
-ocd-division/country:hk/geographical_constituency:new_territories_east,New Territories East
\ No newline at end of file
+id,name,validFrom,validTo
+ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong Island,,2021-06-01

```

Comment:
The only thing unclear to me is whether 2021-06-01 is the correct date for the validTo/From entries. I see the map was published 2021-06-01 for public viewing, but are these districts changed immediately at that point or not until the 2021 election in December: https://en.wikipedia.org/wiki/2021_Hong_Kong_legislative_election

Reply:
Good callout. The new constituencies apply from the next election. Changed the validTo/From dates to the date of the next election, which is 2021-12-19.